### PR TITLE
remove usages of ExecutionContext.Implicits.global

### PR DESCRIPTION
**Pull request by jtjeferreira**
Do I need to maintain backwards compatibility?

### CODE REVIEW

**Comment by adamw** [comment]
@jtjeferreira No, especially since we are before 1.0, backwards compat doesn't need to be maintained. After 1.0, we'll have to be more careful :)

File: server/akka-http-server/src/main/scala/sttp/tapir/server/akkahttp/AkkaHttpServerInterpreter.scala [hunk 1]
```diff
@@ -25,11 +25,11 @@ import sttp.tapir.server.interceptor.reject.RejectInterceptor
 import sttp.tapir.server.interpreter.{BodyListener, FilterServerEndpoints, ServerInterpreter}
 import sttp.tapir.server.model.ServerResponse
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 
 trait AkkaHttpServerInterpreter {
 
-  def akkaHttpServerOptions: AkkaHttpServerOptions = AkkaHttpServerOptions.default
+  def akkaHttpServerOptions: AkkaHttpServerOptions
```
Comment (adamw) [inline server/akka-http-server/src/main/scala/sttp/tapir/server/akkahttp/AkkaHttpServerInterpreter.scala:32]: maybe this should be done as in `PlayServerInterpreter`, with an EC as an implicit parameter in the trait, to maintain consistency across server interpreters

Comment (jtjeferreira) [reply server/akka-http-server/src/main/scala/sttp/tapir/server/akkahttp/AkkaHttpServerInterpreter.scala]: Done

**Comment by jtjeferreira** [comment]
I had to make a few changes in the examples to make them compile. I am not sure if those examples are used in documentation and if thats okay...

**Comment by adamw** [comment]
@jtjeferreira That's fine :) I fixed the docs as well

**Comment by jtjeferreira** [comment]
> @jtjeferreira That's fine :) I fixed the docs as well

Thanks.

**Pull request merged**